packages/frontend/src/AfgHandling.ts

```diff
@@ -43,12 +43,12 @@ export class AfgHandling {
   public receivedFinalized(
     addr: Types.Address,
     finalizedNumber: Types.BlockNumber,
-    finalizedHash: Types.BlockHash,
```

Comment:
Not sure what @maciejhirsz thinks about that one but I find the trailing `,` rather convenient. I would keep them everywhere we can keep them.

Reply:
Ignore that one, my bad.

Reply:
I too like tailing commas, they make git diffs cleaner when adding or removing lines. If there are other things depending on it, I can fix this later so we don't introduce merge conflicts in other open PRs.